src/playlist-loader.js

```diff
@@ -662,6 +665,8 @@ export default class PlaylistLoader extends EventTarget {
       this.trigger('mediachanging');
     }
 
+    this.pendingMedia_ = playlist;
```

Comment:
Added this to prevent switching to the same playlist twice at the start of playback.

Reply:
Where does it get used?

Reply:
`masterPlaylistLoader_.shouldSwitchToMedia_()`